docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -145,16 +154,18 @@ The following example shows how to create a gallery-like flexbox layout using th
     nextButton.Label = labelMap2;
     ```
 
-8.  Add the image to the content area:
+9.  Add the image to the content area:
 
-    ```
+    ```csharp
     /// Add an image to the center of the content area
-    ImageView imageView = new ImageView("image.jpg");
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "image.png");
     imageView.ParentOrigin = ParentOrigin.TopLeft;
     imageView.PivotPoint = PivotPoint.TopLeft;
     content.Add(imageView);
     ```
 
+For more information, see [the full example](download link)
```

Comment:
Now this PR is approved as the comments are applied. However I'm waiting this sample PR is merged and the line 167 is updated with actual sample link. 
As now invalid link is used for the hyperlink, the hyperlink is not working.

![image](https://user-images.githubusercontent.com/9958374/96946312-f8a79f80-151a-11eb-8122-4ec69fc55c40.png)

Reply:
Since PR is approved and it is waiting to be merged only cause of lack of sample link, I will delete it right now and add it later in another PR when sample will be available.